Libplanet/Net/ITransport.cs

```diff
@@ -1,49 +1,176 @@
 using System;
 using System.Collections.Generic;
+using System.Diagnostics.Contracts;
 using System.Threading;
 using System.Threading.Tasks;
 using Libplanet.Net.Messages;
 
 namespace Libplanet.Net
 {
-    internal interface ITransport : IDisposable
+    /// <summary>
+    /// An interface to handle p2p, includes trading <see cref="Message"/>
+    /// and managing <see cref="Peer"/>s. Implements <see cref="IDisposable"/>.
+    /// </summary>
+    public interface ITransport : IDisposable
     {
+        /// <summary>
+        /// <see cref="Peer"/> representation of <see cref="ITransport"/>.
+        /// </summary>
+        [Pure]
         Peer AsPeer { get; }
 
+        /// <summary>
+        /// List of all <see cref="Peer"/>s have in routing table.
+        /// </summary>
+        [Pure]
         IEnumerable<BoundPeer> Peers { get; }
 
+        /// <summary>
+        /// The <see cref="DateTimeOffset"/> of the last message was received.
+        /// </summary>
+        [Pure]
         DateTimeOffset? LastMessageTimestamp { get; }
 
+        /// <summary>
+        /// Setup transport layer.
```

Comment:
How about using third-person grammar in the part? For instance, there is another code written with the format.

I tried to find some documentation in official but I couldn't. But we have used it.

https://github.com/planetarium/libplanet/blob/ddc5e1479c6c9482ad0ffcdd39f31096815dd2da/Libplanet/Store/IStore.cs#L13-L17

Reply:
```suggestion
        /// Initiates transport layer.
```
Your suggestion looks better. Thx